db/user.py

```diff
@@ -44,6 +44,8 @@ def get_by_token(token):
         return dict(row) if row else None
 
 
+
```

Comment:
Be careful about adding things that are not needed. One way to check this is to use a linter, to check the layout of your code, though we don't enforce this at the moment. I often use `git add -p` to verify the diffs which I add, or at the very least always use `git diff` before `git add`, to see the changes that I'll make.